app/controllers/my-projects/assignment/recommendations/add.js

```diff
@@ -343,9 +345,11 @@ export default class MyProjectsProjectRecommendationsAddController extends Contr
             dcpVotingabstainingonrecommendation: this.dispositionForAllActions.dcpVotingabstainingonrecommendation,
             dcpTotalmembersappointedtotheboard: this.dispositionForAllActions.dcpTotalmembersappointedtotheboard,
             dcpConsideration: this.dispositionForAllActions.dcpConsideration,
+            dcpDatereceived: new Date(), // time when user submits recommendation
```

Comment:
This looks unnecessary. On line 352 below, we are already setting `dcpDatereceived` regardless of whether the user has selected "submit by action" or "submit for all actions".

Reply:
This action is getting out of control and we need to save it ASAP